v2/event/event_unmarshal.go

```diff
@@ -416,6 +415,16 @@ func consumeData(e *Event, isBase64 bool, iter *jsoniter.Iterator) error {
 	return nil
 }
 
+func isJson(contentType string) bool {
```

Comment:
Can we combine and reuse this one in the other change you did earlier to keep it consistent and reduce code duplication?